astro/src/content/docs/get-started/run-in-the-cloud/cloud.mdx

```diff
@@ -49,31 +49,36 @@ If you want to use FusionAuth and hand all the management burden to the team who
 
 ### Supported Geographies
 
-FusionAuth Cloud is supported in the following countries:
+FusionAuth has datacenters in the following locations:
 
+Africa
+* South Africa
+
+Asia/Pacific
 * Australia
-* Bahrain
-* Canada
+* Korea
+* Singapore
+
+Europe
 * France
 * Germany
 * Ireland
-* Korea
-* Singapore
-* South Africa
 * Sweden
 * UK
+
+Middle East
+* Bahrain
+
+North America
+* Canada
 * USA
 
-If you would like to have your FusionAuth deployment in a different country, please open a [GitHub issue with details](https://github.com/FusionAuth/fusionauth-issues/issues/).
 
-You can also self-host FusionAuth if you need data residency in a country not listed above.
+If you would like a FusionAuth deployment in a different country, please [contact our sales team](/contact) with your desired location. You can also [download and self-host FusionAuth](/download) if you need data residency in a country not listed above and have the expertise to operate FusionAuth.
```

Comment:
I wouldn't say "country", because that makes it sound like we can deploy anywhere. We can only deploy where AWS has regions. Maybe change that to something like

> If none of those geographical locations meet your needs, please contact our sales team to discuss other options. You can also self-host...

Reply:
Thanks, updated.